src/Core/Silk.NET.BuildTools/Converters/Readers/OpenCLReader.cs

```diff
@@ -1025,14 +1322,49 @@ private static string FormatToken(string token)
             }
 
             var valueString = $"0x{value:X}";
-            var needsCasting = value > int.MaxValue || value < 0;
-            if (needsCasting)
+
+            return valueString;
+        }
+
+        class RenamedEntry
```

Comment:
Can this be made a record struct? This is very verbose.

Reply:
Done